BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/LineItemDetailsViewController.swift

```diff
@@ -55,13 +55,41 @@ class LineItemDetailsViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        let configuration = returnAssistantConfiguration ?? ReturnAssistantConfiguration.shared
-        
-        navigationItem.rightBarButtonItem = UIBarButtonItem(title: .ginibankLocalized(resource: DigitalInvoiceStrings.lineItemSaveButtonTitle),
-                                                            style: .plain,
-                                                            target: self,
-                                                            action: #selector(saveButtonTapped))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            title: .ginibankLocalized(
+                resource: DigitalInvoiceStrings.lineItemSaveButtonTitle
+            ),
+            style: .plain,
+            target: self,
+            action: #selector(saveButtonTapped)
+        )
+        setupView()
+        update()
+        let configuration  = returnAssistantConfiguration ?? ReturnAssistantConfiguration.shared
+        view.backgroundColor = UIColor.from(giniColor: configuration.lineItemDetailsBackgroundColor
+        )
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        guard let lineItem = lineItem, !lineItem.isUserInitiated else { return }
         
+        if isMovingFromParent, transitionCoordinator?.isInteractive == false {
+            /*
+            Automatically save changes before user returns to main screen, so in case he
+            forgets to save it, changes are not lost
+             */
+            proceedWithSaveAction(shouldPopViewController: false)
+        }
+    }
+    
+    private func setupView() {
+        let configuration = returnAssistantConfiguration ?? ReturnAssistantConfiguration.shared
+        if #available(iOS 13.0, *) {
+            view.backgroundColor = .systemBackground
+        } else {
+            view.backgroundColor = .white
+        }
```

Comment:
When we added background color configuration we should have removed these lines. It has no effect as it's overwritten by `configuration.lineItemDetailsBackgroundColor` on line 69 (was on line 240 previously).

You can delete these lines.